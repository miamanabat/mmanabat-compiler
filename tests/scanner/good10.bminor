// this shouldn't care abt keywords here
" what ! auto // oop !"
auto
