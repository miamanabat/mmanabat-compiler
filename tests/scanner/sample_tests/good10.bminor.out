IDENTIFIER                              z
IDENTIFIER                              x
IDENTIFIER                              x
INTEGER_LITERAL                         2
IDENTIFIER                              x
INTEGER_LITERAL                         2
IDENTIFIER                              z
IDENTIFIER                              x
INTEGER_LITERAL                         2
IDENTIFIER                              z
INTEGER_LITERAL                         5
INTEGER_LITERAL                         6
IDENTIFIER                              z
INTEGER_LITERAL                         5
INTEGER_LITERAL                         6
IDENTIFIER                              z
INTEGER_LITERAL                         5
INTEGER_LITERAL                         6
IDENTIFIER                              z
INTEGER_LITERAL                         5
INTEGER_LITERAL                         6
IDENTIFIER                              q
IDENTIFIER                              z
IDENTIFIER                              z
INTEGER_LITERAL                         7
INTEGER_LITERAL                         7
IDENTIFIER                              z
